_includes/index/splash-in-numbers.html

```diff
@@ -3,12 +3,12 @@
     {% assign numbers = site.data.index.numbers %}
     {% for number in numbers %}
     <div class="col-12 col-md-6 col-xl-3 mb-4">
-        <div class="count bg-{{ number.color}} rounded py-4">
+        <div class="count bg-{{ number.color}} rounded py-4 pe-5">
             <i class="{{ number.icon }}"></i>
-            <div class="count-number display-2 text-center text-white">
+            <div class="count-number display-3 text-end text-white">
                 {{ number.number }}
             </div>
-            <div class="count-number-title text-center text-white mt-1">
```

Comment:
This slightly reduces the font of the digits, and aligns them to the right, to make the layout more consistent since the numbers in the footer can be 2, 3 and 4 digits long.